File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/metnorway/locationforecast/LocationForecastMeta.java
package com.lifedawn.bestweather.data.remote.retrofit.responses.metnorway.locationforecast;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LocationForecastMeta {
	@Expose
	@SerializedName("updated_at")
	private String updatedAt;
	
	@Expose
	@SerializedName("units")
	private LocationForecastUnits units;
	
	public String getUpdatedAt() {
		return updatedAt;
	}
	
	public void setUpdatedAt(String updatedAt) {
		this.updatedAt = updatedAt;
	}
	
	public LocationForecastUnits getUnits() {
		return units;
	}
	
	public void setUnits(LocationForecastUnits units) {
		this.units = units;
	}
}

File: app/src/main/java/com/lifedawn/bestweather/data/remote/retrofit/responses/metnorway/locationforecast/LocationForecastMeta.kt
package com.lifedawn.bestweather.data.remote.retrofit.responses.metnorway.locationforecast

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class LocationForecastMeta {
    @Expose @SerializedName("updated_at") var updatedAt: String? = null
    @Expose @SerializedName("units") var units: LocationForecastUnits? = null
}